mappings/net/minecraft/client/performance/MetricSupplier.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5948 net/minecraft/client/performance/MetricSupplier
```

Comment:
Imo this is better named a capturer or factory. The supplier field is only called when the metric result is generated and the supplier's return may change over time.